File: hw/sanchez/u3/hw18OOPConceptsZoo/ZooPython/model/cut.py
class Cut:
    def __init__(self, id: int, description: str, procedure: str, weight: float):
        self.id = id
        self.description = description
        self.procedure = procedure
        self.weight = weight
    
    def __str__(self):
        return f"Cut(id={self.id}, description='{self.description}', procedure='{self.procedure}', weight={self.weight})"

File: ws/elbay/u3/ws24OOPConceptsZoo/OOPConceptsZoo_Java_DidierElbay/OOPConceptsZoo.java/NetBeansProjects/OOPConceptsZoo/src/main/java/ec/edu/espe/oopconceptszoo/model/Cut.java
package ec.edu.espe.oopconceptszoo.model;

/**
 * Represents a meat cut produced from a slaughtered animal.
 *
 * @author Didier Elbay
 */
public class Cut {

    private int id;
    private String description;
    private String procedure;
    private float weight;

    public Cut(int id, String description, String procedure, float weight) {
        this.id = id;
        this.description = description;
        this.procedure = procedure;
        this.weight = weight;
    }

    public int getId()             { return id; }
    public String getDescription() { return description; }
    public String getProcedure()   { return procedure; }
    public float getWeight()       { return weight; }
}
